scripts/post-process.py

```diff
@@ -73,6 +77,29 @@
                 del source[attr]
         filteredSources.append(source)
 
+
+async def get(source, session):
```

Comment:
I'm wondering how sustainable it is to fetch all narinfo at each `sources.json` file generation.
Do you think we could add a caching mecanism? This could come in a followup PR.